CMakeLists.txt

```diff
@@ -105,12 +120,12 @@ if(NOT CMAKE_DISABLE_FIND_PACKAGE_CUDA AND
   check_language(CUDA)
```

Comment:
I cannot comment on lines 118-119: "Since the OPTIONAL flag of enable_language is currently ignored CMake will error out below if the options do not work."
Would this be outdated then?